src/dtx/dtx_rpc.c

```diff
@@ -1101,7 +1101,7 @@ dtx_refresh_internal(struct ds_cont_child *cont, int *check_count, d_list_t *che
 				if (rc1 != DTX_ST_COMMITTED && rc1 != DTX_ST_ABORTED &&
 				    rc1 != -DER_NONEXIST) {
 					if (!for_io)
-						D_INFO("Hit some long-time DTX "DF_DTI", %d\n",
+						D_WARN("Hit some long-time DTX "DF_DTI", %d\n",
```

Comment:
does this really need to be a warning?
everything about his PR is making things debug, but this is going the other way :-)

Reply:
The long-time DTX is unexpected, there may be some bad things happened, generate warning message to reminder the user.